snuba/api/query.py

```diff
@@ -136,32 +146,59 @@ def raw_query(
 
                     except BaseException as ex:
                         error = str(ex)
-                        status = 500
                         logger.exception("Error running query: %s\n%s", sql, error)
+                        stats = log_query_and_update_stats(request, sql, timer, stats, 'error', query_settings)
+                        meta = {}
                         if isinstance(ex, ClickHouseError):
-                            result = {
-                                "error": {
-                                    "type": "clickhouse",
-                                    "code": ex.code,
-                                    "message": error,
-                                }
-                            }
+                            type = "clickhouse"
```

Comment:
Aren't you reassigning the class `type` this way ?
```
Python 3.7.4 (default, Sep  7 2019, 18:27:02) 
[Clang 10.0.1 (clang-1001.0.46.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> type
<class 'type'>
>>> type = "a"
>>> type
'a'
```

Reply:
Yeah, I typically don't worry about this too much but happy to change it.